### DISCUSSION TITLE
New Chromium Update Messes Up Background

### DISCUSSION THREAD

**Discussion by Niferu** [tailwindlabs/tailwindcss#12361] [Help] [answered]
Hello Everyone,

I have recently build my portfolio website with tailwind and I have used below code to create mixed background:

```typescript
import "@/styles/globals.css"
import ActiveSectionContextProvider from "@/context/active-section-context"
import Header from "@/components/header"
import Footer from "@/components/footer"
import ThemeSwitch from "@/components/theme-switch"
import ThemeContextProvider from "@/context/theme-context"
import { Montserrat } from "next/font/google"
import { Toaster } from "react-hot-toast"

const montserrat = Montserrat({ subsets: ["latin"] })

export const metadata = {
    title: "Niferu Portfolio",
    description: "Portfolio",
}

export default function RootLayout({ children }: { children: React.ReactNode }) {
    return (
        <html lang="en" className="!scroll-smooth">
            <body className={`${montserrat.className} bg-fixed pt-28 sm:pt-36 bg-gray-300 dark:bg-devil`}>
                <div className="bg-fixed bg-[#fbe2e3] dark:bg-[#671f91a8] absolute top-[1rem] -z-10 right-[11rem] h-[31.25rem] w-[31.25rem] rounded-full blur-[10rem] sm:w-[68.75rem]"></div>
                <div className="bg-fixed bg-[#dbd7fb] dark:bg-[#48457c] absolute top-[9rem] -z-10 left-[-35rem] h-[31.25rem] w-[50rem] rounded-full blur-[10rem] sm:w-[68.75rem] md:left-[-33rem] lg:left-[-28rem] xl:left-[-15rem] 2xl:left-[-5rem]"></div>

                <ThemeContextProvider>
                    <ActiveSectionContextProvider>
                        <Header />
                        {children}
                        <Footer />

                        <Toaster
                            position="top-right"
                            containerStyle={{
                                top: "5rem",
                            }}
                        />
                        <ThemeSwitch />
                    </ActiveSectionContextProvider>
                </ThemeContextProvider>
            </body>
        </html>
    )
}
```

Effect can be found here: https://niferu.vercel.app/

Now if you toggle dark mode you will see below barely visible lines:
![image](https://github.com/tailwindlabs/tailwindcss/assets/110537615/1f203d9f-5f37-42f4-9f47-169b5973e6c7)

The problem is that those appeared after yesterday chromium update and until that there was no strange barely visible lines... Page is working fine on safari for example, but chrome/brave are broken since yesterday and Edge is broken since today...

Does anyone know how can I fix that issue? I believe it is because I'm using 2 div elements, but tbh I have no idea how to reach same effect without those... Any help would be much appreciated, thanks!

Additionally my globals.css

```css
@tailwind base;
@tailwind components;
@tailwind utilities;

html {
    --line-color: rgba(67, 52, 28, 1);
}

html.dark {
    --line-color: #805d39;
}

@layer utilities {
    .borderDevil {
        @apply border dark:border-orange-400/10;
    }

    .borderBlack {
        @apply border-black/10;
    }
}
```

**Comment by Niferu** [comment]
I will add that previously, before chrommium update it was looking like below:

![image](https://github.com/tailwindlabs/tailwindcss/assets/110537615/b26e69ab-678e-4275-906e-df66fcafabee)

As you can see no strange barely visible lines. Does anyone have idea how to fix it?

**Comment by CharonTom** [comment]
Hi, I also have the same problem and it's definitely from the last chrome update. Every website I know which use this kind of blur design for backgrounds has the same issues.

**Reply by Niferu** [reply]
@CharonTom 

Thanks for letting me know I'm not alone. I have figured out the same it is coming from blur effect, found 2 posts on chromium regarding this "bug" [first](https://bugs.chromium.org/p/chromium/issues/detail?id=1499791&q=background&can=2) and [second](https://bugs.chromium.org/p/chromium/issues/detail?id=1499528#c1). Should we wait for correction or anyone know some trick or workaround to get same effect without lines?

**Comment by CharonTom** [comment]
I can't find a similar solution.  Hopefully they'll fixed it or I'll have to find another background design for my portoflio.

**Comment by Niferu** [comment]
Semms like it got confirmed as real bug and should be fixed in incoming update. [LINK](https://bugs.chromium.org/p/chromium/issues/detail?id=1500021)

**Comment by Niferu** [comment, marked as answer]
Hi @CharonTom 

Adding the code below to div with blur effect will fix the issue:
```js
style={{ transform: "translate3d(0,0,0)" }}
```

For example in my case it was:
```js
<div
   style={{ transform: "translate3d(0,0,0)" }}
   className="bg-[#fbe2e3] dark:bg-[#671f91a8] absolute top-[1rem] -z-10 right-[11rem] h-[31.25rem] w-[31.25rem] rounded-full blur- 
   [10rem] sm:w-[68.75rem]"
></div>

<div
   style={{ transform: "translate3d(0,0,0)" }}
   className="bg-[#dbd7fb] dark:bg-[#48457c] absolute top-[9rem] -z-10 left-[-35rem] h-[31.25rem] w-[50rem] rounded-full blur- 
  [10rem] sm:w-[68.75rem] md:left-[-33rem] lg:left-[-28rem] xl:left-[-15rem] 2xl:left-[-5rem]"
></div>
```

**Reply by CharonTom** [reply]
Thank you very much ! It will be usefull until they patch the issue.
